classes/sdcard_image-openvario.bbclass

```diff
@@ -15,11 +15,43 @@ inherit image_types
 
 # This image depends on the rootfs image
 IMAGE_TYPEDEP_openvario-sdimg = "${SDIMG_ROOTFS_TYPE}"
-
 # Boot partition volume id, not longer then 11 character
-BOOTDD_VOLUME_ID ?= "${MACHINE#*-}"
 
+IMAGE_CMD_prepend () {
+	case ${MACHINE} in
+	  "openvario-7-CH070")
+	    BOOTDD_VOLUME_ID="OV-CH070"
+	    ;;
+	  "openvario-7-CH070-DS2")
+	    BOOTDD_VOLUME_ID="OV-CH070DS2"
+	    ;;
+	  "openvario-7-AM070")
+	    BOOTDD_VOLUME_ID="OV-AM070"
+	    ;;
+	  "openvario-7-AM070-DS2")
+	    BOOTDD_VOLUME_ID="OV-AM070DS2"
+	    ;;
+	  "openvario-7-PQ070")
+	    BOOTDD_VOLUME_ID="OV-PQ070"
+	    ;;
+	  "openvario-57-lvds")
+	    BOOTDD_VOLUME_ID="OV-CH057"
+	    ;;  
+	  "openvario-57-lvds-DS2")
+	    BOOTDD_VOLUME_ID="OV-CH057DS2"
+	    ;;   
+	  "openvario-43-rgb")
+	    BOOTDD_VOLUME_ID="OV-RGB43"
+	    ;;
+	  *)
+	    BOOTDD_VOLUME_ID="OV-Unknown"
+	    ;;
+	esac
+}
 
+#### Boot partition volume id, not longer then 11 character
+###BOOTDD_VOLUME_ID ?= "${MACHINE#*-}"
+#
```

Comment:
Because I prepare this sdcards for several 'machines' I prefere a solutione with different volume names - because I want to see very easy what is in. I like much more to have the description of version too, but 11 characters allowed only a small name ;-( 
But the 'openvario-7' says nothing to me (ok, it shows, this is a OV card...)!
Another proposal would be to define a variable (maybe with placeholder) in a config script or in the envorinment ...?

Reply:
I agree with @kedder . Have a specific name for each device SDCard is fine for me, but lets contain all the variables in less locations as possible. 
Maybe in the config file of the machine ??

Reply:
How about adding a constant to `conf/machine/openvario-57-lvds.conf` and friends? I suspect we can just define `BOOTDD_VOLUME_ID` straight in these machine files. I feel it is more in line with openembedded-way of doing things.

Reply:
This I will do later - so I change in the commit with a fixed label to avoid the baking error ( not more then 11 characters allowed) - and maybe we will discuss a better name in the #148 issue?